sentry_sdk/tracing.py

```diff
@@ -435,6 +457,15 @@ def __init__(
                 "instead of Span(transaction=...)."
             )
             name = kwargs.pop("transaction")
+
+        if "_span_id_generator" in kwargs:
+            raise TypeError("Argument _span_id_generator can only be provided to Spans")
+
+        # next() on itertools.count() is a way to get-and-increment an integer
+        # "atomically" on Python runtimes with a GIL
+        if _ITERTOOLS_COUNT_IS_ATOMIC and kwargs.pop("_fast_span_ids", None):
+            kwargs["_span_id_generator"] = itertools.count()
```

Comment:
We should start from an offset != 0 to avoid using the same id for `trace_id` and `span_id`, if anything for sanity's sake.

![image](https://user-images.githubusercontent.com/88819/90124670-14473a80-dd61-11ea-9a0a-ff11812c6801.png)